python/ql/test/library-tests/frameworks/bottle/taint_test.py

```diff
@@ -0,0 +1,21 @@
+import bottle
+from bottle import response, request
+
+
+app = bottle.app()
```

Comment:
I am not sure this works, where did you find this? I think the idiomatic example is here: https://bottlepy.org/docs/dev/tutorial.html#the-application-object

Reply:
You can find it in some of the [documentation](https://bottle.readthedocs.io/en/stable/api.html#bottle.app). There are also a lot of examples on Github search.

Reply:
Ok, that is great to hear :-) Should we also test that we support the simple example from the tutorial, though?

Reply:
Added